demo/site-pages/src/config.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-console */
+/* eslint-disable no-restricted-syntax */
```

Comment:
for site-pages this rule should be disabled for the whole site. And you should mention in the changeset (or even migration guide) that for projects using the old build that this rule should also be disabled there.